server/src/controllers/Events/resolver.ts

```diff
@@ -795,17 +770,32 @@ ${unsubscribeOptions}`,
     });
     await deleteEventReminders(id);
 
+    const chapterURL = `${process.env.CLIENT_LOCATION}/chapters/${event.chapter.id}`;
+    const eventURL = `${process.env.CLIENT_LOCATION}/events/${event.id}`;
```

Comment:
the test is fallin because these two don't have unsubscribe in them, I think I need to add string that activate unsubscribe commands 🤔

Reply:
:thinking: This doesn't seem related to that test.

I think the `getUnsubscribeOptions` is returning tokens wrapped in an `<a>` instead of proper links.